### PR TITLE
chore(monorepo): remove jest

### PR DESCRIPTION
**Pull request by koji**
# Overview
remove jest and add jsdom v16.
Updating jsdom version will be done in a following PR.

close AUTH-83

# Test Plan
1. `make teardown-js && make setup-js`
2. check tests-js, check-js, and  lint-js work properly 

# Changelog
- remove jest and configs that are related to jest
- add jsdom

# Review requests

# Risk assessment
low

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -107,31 +107,21 @@ module.exports = {
...
         'scripts/*.@(js|ts|tsx)',
       ],
-      env: {
-        jest: true,
-      },
-      extends: ['plugin:jest/recommended'],
+      // env: {
+      //   jest: true,
+      // },
+      // extends: ['plugin:jest/recommended'],
```
Comment (mjhuff) [inline .eslintrc.js]: I think it's better just to delete this.

File: .eslintrc.js [hunk 2]
```diff
@@ -107,31 +107,21 @@ module.exports = {
...
-        'jest/valid-title': 'warn',
-        'jest/no-conditional-expect': 'warn',
-        'jest/no-alias-methods': 'warn',
-        'jest/valid-describe-callback': 'warn',
       },
     },
     {
       files: ['**/__tests__/**test.tsx'],
-      env: { jest: true },
+      // env: { jest: true },
```
Comment (mjhuff) [inline .eslintrc.js]: Same

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -291,7 +291,7 @@ JavaScript dependencies are installed by [yarn][]. When calling yarn, you should
 A development dependency is any dependency that is used only to help manage the project. Examples of development dependencies would be:
 
 - Build tools (webpack, babel)
-- Testing/linting/checking tools (jest, typescript, eslint)
+- Testing/linting/checking tools (vitest, typescript, eslint)
```
Comment (mjhuff) [inline CONTRIBUTING.md:294]: Nice!

File: app/src/atoms/Chip/index.tsx [hunk 4]
```diff
@@ -81,6 +81,7 @@ export function Chip({
     background === false && type !== 'basic'
       ? COLORS.transparent
       : CHIP_PROPS_BY_TYPE[type].backgroundColor
+  console.log('backgroundColor', backgroundColor)
```
Comment (mjhuff) [inline app/src/atoms/Chip/index.tsx]: Stray console log?

File: package.json [hunk 5]
```diff
@@ -108,7 +105,7 @@
     "handlebars-loader": "^1.7.1",
     "html-webpack-plugin": "^3.2.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^26.6.3",
+    "jsdom": "^16.4.0",
```
Comment (mjhuff) [inline package.json:108]: Why do we need this package now?

**Review by mjhuff — COMMENTED** [review]
Thanks for tackling this! Left a few clean up comments. Just need some clarification around the jsdom package, too.

File: app/src/atoms/Chip/index.tsx (hunk 4 above)
Comment (koji) [reply app/src/atoms/Chip/index.tsx]: good catch, I will delete that.

File: package.json (hunk 5 above)
Comment (koji) [reply package.json:108]: The entire tests fail without installing `jsdom` and the vitest prompted "install jsdom?"

Comment (mjhuff) [reply package.json:108]: Gotcha - looks like jsdom is a dependency of jest, so that's why it was working before.

**Review by mjhuff — APPROVED** [review]
LGTM when CI passes!

File: package.json (hunk 5 above)
Comment (koji) [reply package.json:108]: yup. I thought vitest would do the same thing for us but not...

**Pull request merged**